scripts/fetch/fetch.precip_spatial.R

```diff
@@ -0,0 +1,64 @@
+fetch.precipSpatial <- function(viz = as.viz('precip-spatial')){
+
+  view <- readDepends(viz)[["view-limits"]]
+  
+  ### constants
+  cell_size <- .1 # degrees -- set to make grid more course or dense.
+  states <- c("florida", "georgia", "alabama", "south carolina") # from maps states id list.
```

Comment:
to do this I think you want to depend on the other states that come in through fetch, otherwise, this isn't as re-usable.

Reply:
I use this for subsetting the ones that come in from the rest of the package. Is there a standard subset somewhere else in the package?